### ISSUE TITLE
Loss stays the same, only PSL decreases, and the result is all zero

### ISSUE THREAD

**Issue by kyoungrok0517** [harsh19/SPINE#6]
Hi, I'm mostly following the suggested hyperparameter settings except the batch_size (512) and epoch (1000). What I observe is at some point `reconstruction loss` isn't improving but only PSL decreases. I thought this is expected phenomena since sparsity loss get's better as iteration goes. But what I see from the result is all-zero vectors even for the words that should convey important meanings (e.g. "cat"). Is there optimal point that I should stop iteration?

** One more question: Why is ASL always zero in my training? I'm training on fasttext embeddings.

**Comment by harsh19** [comment]
Hi,
Sorry but we haven't experimented with fasttext - you should try out different hyperparameters to find what works best. We have provided some intuitions about hyperparams such as hidden layer size in our paper, which can be of some guidance on what range of values seem to work.
